frontend/packages/console-app/src/components/cloud-shell/setup/NamespaceSection.tsx

```diff
@@ -58,17 +60,24 @@ const NamespaceSection: React.FC<NamespaceSectionProps> = ({ flags }) => {
           canCreate
             ? [
                 {
-                  actionTitle: 'Create Project',
+                  actionTitle: t('cloudshell~Create Project'),
```

Comment:
am not sure for this and Project, but this can be common in other namespace. So shall we use the same

Reply:
There are ongoing discussions about having frequently used words in the public namespace, I can move these once that is finalized.

Reply:
I believe the public words will be a generated list based on N number of occurrences.